math/finding_number_of_Digits_in_a_Number.cpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * @author [ANSHUMAAN](https://github.com/amino19)
+ * @file [Program to count digits in an
+ * integer](https://www.geeksforgeeks.org/program-count-digits-integer-3-different-methods)
+ *
+ * @brief Finding number of Digits in a Number
+ * @details It is a very basic math of finding number of digits in a given
+ * number i.e, we can use it by inputting values whether it can be a
+ * positive/negative value, lets say integer. There is also second method to do
+ * it with, by using "K = floor(log10(N) + 1)", but its only applicable for
+ * numbers (not integers).
+ * For more details, refer Algorithms-Explanation
```

Comment:
Once your PR https://github.com/TheAlgorithms/Algorithms-Explanation/pull/101 is merged, this link will work properly.
```suggestion
 * For more details, refer to the [Algorithms-Explanation](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/en/Basic%20Math/Finding the number of digits in a number.md) repository.
```